src/pluginSystem/handlePluginData.js

```diff
@@ -14,26 +14,24 @@
  * You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
 
-
 const fs = require("fs");
 
 const PluginSystem = require("./pluginSystem.js");
-
+const { logger } = require("output-logger/lib/logger.js");
```

Comment:
This import is wrong, it will bypass the log hold back feature. Please use the global logger function or use `controller.logger`